.circleci/workflows.yml

```diff
@@ -138,6 +140,8 @@ jobs:
         # to CI beforehand, for manual inspection in case of failures.
         ./x.py --stage 2 dist $(ferrocene/ci/split-tasks.py dist:docs)
         ./x.py --stage 2 test $(ferrocene/ci/split-tasks.py test:docs)
+        # Build the API docs of the certified subset of libcore
+        ./x.py doc library/core --set rust.std-features='["ferrocene_certified"]'
```

Comment:
This job creates a cache that all future jobs use, so I'm not sure this accomplishes what you want. I suspect that this will either damage or invalidate the cached libcore, possibly causing problems or making the build process longer.

I recommend removing this for now. Once we land custom targets we can add it correctly.